### ISSUE TITLE
PHP-redis module is not working with centos 7

### ISSUE THREAD

**Issue by hardyjohnson** [copious-cookbooks/php#17]
```
==> web:   * execute[enable_php_redis] action run
==> web:
==> web:
==> web:     ================================================================================
==> web:     Error executing action `run` on resource 'execute[enable_php_redis]'
==> web:     ================================================================================
==> web:
==> web:     Errno::ENOENT
==> web:     -------------
==> web:     No such file or directory - phpenmod
==> web:
==> web:     Resource Declaration:
==> web:     ---------------------
==> web:     # In /var/chef/cache/cookbooks/cop_php/recipes/redis.rb
==> web:
==> web:      34: execute 'enable_php_redis' do
==> web:      35:     action      :run
==> web:      36:     user        'root'
==> web:      37:     command     'phpenmod redis'
==> web:      38:     notifies    :restart, "service[#{node['php']['sapi']['fpm']['fpm_service_name']}]", :delayed
==> web:      39: end
```

**Comment by hardyjohnson** [comment]
This is resolved by #18

**Comment by hardyjohnson** [comment]
Merged, closing issue.
